adapters/base-spring/pom.xml

```diff
@@ -26,6 +26,10 @@
   <description>Defines common dependencies for Spring-based protocol adapter implementations</description>
 
   <dependencies>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-adapter-base-spring</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-service-base-spring</artifactId>
```

Comment:
Wouldn't it be enough to have just `hono-adapter-base-spring` here? (see also other comment)

Reply:
In general, yes. However, the Spring based adapter modules have explicit dependencies on classes in hono-service-base-spring, which is what I want to make explicit here.